app/src/main/res/layout/dashboard_categories_concat.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
there is no reason to include `concat` in the layout name, you also aren't including the fact that this is part of the dashboard tools UI.